Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Build the manager binary
+FROM golang:1.16 as builder
```

Comment:
Nit: I'd use always latest (1.17). We can do it in future commits.

Reply:
ok we can do that in next PRs